### PR TITLE
ES6 modules ftw

### PR DESCRIPTION
**Pull request by cmisenas**
So a couple of things:
- I chose to keep the current setup where we have a global Backbone and Marionette object because on some places we use Backbone.Marionette.
- I also don't know why `this.sinon` all of a sudden stopped working (has to be just `sinon` now)?
- We need to figure out how to fix the `<%= this.version =>`
- Also, couldn't figure out but why wasn't this failing before? (i.e., was there ever a `Marionette.buildRegion`?)                                                                                                                                                                               https://github.com/marionettejs/backbone.marionette/blob/d182dc3f09b13a2603c142029d45994e1455105c/test/unit/region.build-region.spec.js#L261

### CODE REVIEW

File: test/unit/region.build-region.spec.js [hunk 1]
```diff
@@ -276,7 +274,7 @@ describe('Region', function() {
             myOtherRegionOption: 'foobar'
           };
 
-          this.region = Marionette.buildRegion(this.definition, this.DefaultRegionClass);
+          this.region = Marionette.Region.buildRegion(this.definition, this.DefaultRegionClass);
```
Comment (jasonLaster) [inline test/unit/region.build-region.spec.js:277]: wow - nice catch

File: test/unit/setup/node.js [hunk 2]
```diff
@@ -2,9 +2,7 @@ var chai = require('chai');
 var sinon = require('sinon');
 var sinonChai = require('sinon-chai');
 var chaiJq = require('chai-jq');
-var requireHelper = require('./require-helper');
-import {setEnabled} from '../../../src/features';
-
+import Marionette from '../../../src/backbone-marionette';
```
Comment (jasonLaster) [inline test/unit/setup/node.js:5]: beautiful

**Comment by jasonLaster** [comment]
yea, this.version is legit

funny on buildRegion. oops

**Comment by cmisenas** [comment]
Hah @jasonLaster. Could not figure that one as well. :P

**Pull request merged**